app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# User class
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
do you need all these devise modules? if not, remove the ones you're going to use